rush.json

```diff
@@ -862,12 +862,38 @@
       "reviewCategory": "tests",
       "shouldPublish": false
     },
+    /*
     {
       "packageName": "install-test-workspace",
       "projectFolder": "build-tests/install-test-workspace",
       "reviewCategory": "tests",
       "shouldPublish": false
     },
+    */
+    {
+      "packageName": "rush-lib-test",
+      "projectFolder": "build-tests/install-test-workspace/rush-lib-test",
```

Comment:
Should these go in a different top-level folder, and we keep the max depth at 2? @octogonz

Reply:
Yes, maybe it could be called `build-tests-subspace` alongside `build-tests` and `build-tests-samples`.